hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -630,6 +630,15 @@
       clock_group: "infra",
       reset_connections: {rst_ni: "sys"},
       base_addrs: {rom: "0x00008000", regs: "0x411e0000"}
+      // Memory regions must be associated with a dedicated
+      // TL-UL device interface.
```

Comment:
I'm not sure this is the right place to document this requirement. Should we be adding some check for this in the topgen code?

Reply:
Yes I can put in an additional topgen check for this.